From b81adc9f48a052ba0ad741bd8637804ff959eb03 Mon Sep 17 00:00:00 2001
From: Tushar Dave <tushar.n.dave@intel.com>
Date: Fri, 1 Aug 2014 15:29:55 -0700
Subject: [PATCH 1/1] ft5x0: touchscreen driver suspend fail hacked

After one suspend and resume , touch on ecs fails to suspend.
This is a test hack to see if suspned again and work.

Signed-off-by: Tushar Dave <tushar.n.dave@intel.com>
---
 drivers/input/touchscreen/ft5x06_ts.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/input/touchscreen/ft5x06_ts.c b/drivers/input/touchscreen/ft5x06_ts.c
index 86afea75994f..d9124c468741 100644
--- a/drivers/input/touchscreen/ft5x06_ts.c
+++ b/drivers/input/touchscreen/ft5x06_ts.c
@@ -753,13 +753,13 @@ static int ft5x0x_ts_resume(struct device *dev)
 
 	dev_dbg(&tsdata->client->dev, "resume");
 
+	ft5x0x_wake_device(tsdata);
+
 	ret = ft5x0x_set_pmode(tsdata, PMODE_ACTIVE);
 
 	if (ret < 0)
 		return ret;
 
-	ft5x0x_wake_device(tsdata);
-
 	enable_irq(tsdata->irq);
 
 	return 0;
-- 
1.7.9.5

